src/options.c

```diff
@@ -535,11 +535,11 @@ static void showVersion(void)
 
 static char *optionsNameThumbnail(const char *name)
 {
-    const size_t nameLength = strlen(name);
+    const ptrdiff_t nameLength = (ptrdiff_t)strlen(name);
```

Comment:
Let the assignment do the equivalent of a cast unless that also produces a warning. It's the same issue as using the type and not the variable name in sizeof's operand.

Reply:
> Let the assignment do the equivalent of a cast unless that also produces a warning.

Doesn't seem to produce a warning as for now at least. (and if it did, then we'd probably just have to switch to size_t (since that's that the `stream*` functions accept as param) or just disable the warning).